charts/sonarqube-lts/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: sonarqube
 description: SonarQube is an open sourced code quality & code security scanning tool
```

Comment:
```suggestion
description: SonarQube offers Code Quality and Code Security analysis for up to 27 languages. Find Bugs, Vulnerabilities, Security Hotspots and Code Smells throughout your workflow.
```

Reply:
FYI I keep editing out "is an open source scanning tool" because we're covering 3 editions here & only one of them is OSS